GetIntoTeachingApiTests/Services/CrmServiceTests.cs

```diff
@@ -517,6 +529,19 @@ public void GetTeachingEvent_WhenTeachingEventDoesNotExist_ReturnsNull()
             result.Should().BeNull();
         }
 
+        [Fact]
+        public void GetTeachingEvent_CallLoadProperty()
+        {
+            _mockService.Setup(mock => mock.CreateQuery("msevtmgt_event", _context))
+                .Returns(MockTeachingEvents());
+
+            var result = _crm.GetTeachingEvent("event_one");
+
+            result.ReadableId.Should().Be("event_one");
+            _mockService.Verify(mock => mock.LoadProperty(It.IsAny<Entity>(),

```

Comment:
I'd be inclined to roll this into the existing `GetTeachingEvent_WhenTeachingEventExists_ReturnsTeachingEvent` test